packages/mdc-fab/mdc-fab.scss

```diff
@@ -12,98 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@import "@material/animation/functions";
-@import "@material/animation/variables";
-@import "@material/elevation/mixins";
-@import "@material/ripple/mixins";
-@import "@material/theme/mixins";
+@import "@material/theme/variables";
+@import "./mixins";
 
 // postcss-bem-linter: define fab
-.mdc-fab {
-  @include mdc-ripple-base;
-  @include mdc-ripple-bg((pseudo: "::before", base-color: white, opacity: .16));
-  @include mdc-ripple-fg((pseudo: "::after", base-color: white, opacity: .16));
-
-  display: inline-flex;
-  position: relative;
-  justify-content: center;
-  width: 56px;
-  height: 56px;
-  padding: 0;
-  transition: box-shadow 280ms $mdc-animation-standard-curve-timing-function;
-  border: none;
-  border-radius: 50%;
-  cursor: pointer;
-  user-select: none;
-  box-sizing: border-box;
-  fill: currentColor;
-  -moz-appearance: none;
-  -webkit-appearance: none;
-  overflow: hidden;
-
-  &:not(.mdc-ripple-upgraded) {
-    -webkit-tap-highlight-color: rgba(0, 0, 0, .18);
-  }
-
-  @include mdc-theme-prop(background-color, secondary);
-  @include mdc-theme-prop(color, text-primary-on-secondary);
-  @include mdc-elevation(6);
 
-  &--mini {
-    width: 40px;
-    height: 40px;
-  }
-
-  &--plain {
-    background-color: white;
-
-    @include mdc-theme-prop(color, text-primary-on-light);
-    @include mdc-ripple-bg((pseudo: "::before"));
-    @include mdc-ripple-fg((pseudo: "::after"));
-  }
-
-  &:active,
-  &:focus {
-    outline: none;
-  }
+$mdc-theme-property-values-custom: map-merge(
+  $mdc-theme-property-values-custom, (
+    // Default
+    fab-default-tap-highlight-color: rgba(0, 0, 0, .18),
+    // Plain
+    fab-plain-bg-color: white,
+    fab-plain-tap-highlight-color: rgba(0, 0, 0, .18),
+    // Disabled
+    fab-disabled-bg-color: rgba(0, 0, 0, .12),
+    fab-disabled-fg-color: rgba(0, 0, 0, .26)
+  )
+);
 
-  &:active {
-    @include mdc-elevation(12);
-  }
-
-  &:hover {
-    cursor: pointer;
-  }
-
-  &::-moz-focus-inner {
-    padding: 0;
-    border: 0;
+.mdc-fab {
+  @include mdc-fab-base;
+  @include mdc-fab-theme((
+    bg-color: secondary,
+    fg-color: text-primary-on-secondary,
+    ripple-config: (base-color: white, opacity: .16),
+    tap-highlight-color: fab-default-tap-highlight-color
+  ));
+
+  :disabled &,
```

Comment:
I think we probably want users to be able to customize disabled colors. If their theme happens to look similar-ish to our default disabled colors, they would probably want to change disabled to differentiate it from enabled.